### PR TITLE
TierSelector: add product variant descriptions

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

`TierSelector`: add product variant descriptions

## Justify why they are needed

This should ideally come from the API but that's not possible in the short term. Added comment to remember.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1564** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1564" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1564?utm_source=stack-comment).

File: apps/store/src/components/TierSelector/TierSelector.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const TierItem = ({
           <PriceText isSelected={isSelected}>{price}</PriceText>
         </TitleItem>
       </TitleContainer>
-      <SecondaryText>{description}</SecondaryText>
+      {description && <SecondaryText>{description}</SecondaryText>}
```
Comment (alebedev) [inline apps/store/src/components/TierSelector/TierSelector.tsx:46]: Nit: How would suggested item + description looks like?  Would there be any overflow / overlap / other visual artifacts?

**Review by alebedev — APPROVED** [review]
Nice!

File: apps/store/src/components/TierSelector/TierSelector.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/TierSelector/TierSelector.tsx:46]: Hmm no don't think so but good point -- the design assumes we always have description so we are a little on our own now

Also "suggested text" is an Insurely feature :P

**Pull request merged**